packages/protvista-structure/package.json

```diff
@@ -25,11 +29,14 @@
   "author": "xwatkins@ebi.ac.uk",
   "license": "MIT",
   "dependencies": {
-    "lodash-es": "^4.17.11",
-    "whatwg-fetch": "^3.4.1"
-  },
-  "peerDependencies": {
-    "litemol": "^2.4.2"
+    "data-loader": "^3.0.0",
+    "molstar": "2.0.6",
+    "react": "17.0.2",
+    "whatwg-fetch": "3.4.1"
```

Comment:
As we have a dependency on `data-loader`, might as well use it for the fetch too and not depend on `whatwg-fetch`?

Reply:
Done